### ISSUE TITLE
Keyboard focus is not visible at Hamburger menu present at the UI.

### ISSUE THREAD

**Issue by msftedad** [docsifyjs/docsify#2310]
#### Steps to reproduce

1. Open the PWA Builder URL-https://docs.pwabuilder.com/#/ in Edge browser.
2. PWA Builder home page will appear.
3. Navigate on the Hamburger menu with keyboard.
4. Observe the Keyboard focus.

#### Current behaviour
Keyboard focus is not visible at Hamburger menu present at the UI.

**Attachments:**

![Keyboard focus is not visible at Hamburger menu present at the UI](https://github.com/docsifyjs/docsify/assets/93735775/07b6d273-c840-4a7a-933e-8deb338cfd99)

https://github.com/docsifyjs/docsify/assets/93735775/05bdfa11-53b8-4113-907e-a454834125ac

#### Expected behaviour

Proper keyboard focus must be visible for the Hamburger menu present at the UI.

#### Other relevant information

- Docsify version: 4.4.4

- [ X] Bug still occurs when all/other plugins are disabled?

- Docsify plugins (if the bug happens when plugins enabled, please try to isolate the issue): NA

#### Please create a reproducible sandbox
NA

#### Mention the docsify version in which this bug was not present (if any)
NA

**Comment by jhildenbiddle** [comment]
This is a theme-related issue. Please recreate this issue in the [docsify-themeable](https://github.com/jhildenbiddle/docsify-themeable) repo.
